### PR TITLE
Added color variables

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

**Comment by shruthipatil11** [comment]
Looks great.
Semantic html could be used like main, header, section , footer , figure etc

**Review by SinchanaMS — APPROVED** [review]
Everything looks good!

File: common.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+body
+{
+    background-color: var(--main-background-color);
+    box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
+    font-family: Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
+}
+*
+{
+    padding: 0;
+    margin: 0;
```
Comment (ashishkatiyar2404) [inline common.css]: you can add box-sizing : border-box property here

**Review by ashishkatiyar2404 — APPROVED** [review]
your code look cleans and looks good to merge💯

File: common.css (hunk 1 above)
Comment (ShrutiLakhani) [reply common.css]: Done

**Pull request merged**